### PR TITLE
Added temp QML view with icons and some api changes

### PR DESCRIPTION
**Pull request by Zarrett**
1.Remade weatherdata class for using it in a list of qml properties
2.Added temporary view and icons for daily forecast
3.Made some api changes to wotk with new weatherdata class
4.fixed some typos in api

### CODE REVIEW

File: weather/ui/WeatherIcon.qml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                case "13n":
+                    "../../icons/weather-snow.png"
+                    break;
+                case "50d":
+                case "50n":
+                    "../../icons/weather-fog.png"
+                    break;
+                default:
+                    "../../icons/weather-unknown.png"
+                }
```
Comment (e1sem) [inline weather/ui/WeatherIcon.qml]: If I am not mistaken you will get a warning here due to the missed `break`

File: weather/ui/ForecastIcon.qml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        anchors.topMargin: top.height / 5 - dayText.paintedHeight
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    WeatherIcon {
+        id: icon
+        weatherIcon: top.weatherIcon
+
+        property real side: {
+            var h = 3 * top.height / 5
```
Comment (e1sem) [inline weather/ui/ForecastIcon.qml]: Very informative name

**Comment by e1sem** [comment]
> 1.Remade weatherdata class for using it in a list of qml properties
> 2.Added temporary view and icons for daily forecast
> 3.Made some api changes to wotk with new weatherdata class
> 4.fixed some typos in api

**Comment by e1sem** [comment]
Also i don't think that you need to stage `MeteoPrototype.pro.user` and `MeteoPrototype.pro.user.d1df319.4.9-pre1`

File: MeteoPrototype.pro [hunk 3]
```diff
@@ -44,11 +45,11 @@ qnx: target.path = /tmp/$${TARGET}/bin
...
-copydata.commands = $(COPY_DIR) $$PWD/settings.conf $$OUT_PWD
-first.depends = $(first) copydata
-export(first.depends)
-export(copydata.commands)
-QMAKE_EXTRA_TARGETS += first copydata
+#copydata.commands = $(COPY_DIR) $$PWD/settings.conf $$OUT_PWD
+#first.depends = $(first) copydata
+#export(first.depends)
+#export(copydata.commands)
+#QMAKE_EXTRA_TARGETS += first copydata
```
[the remark this answers is not in the document]
Comment (ValentiWorkLearning) [reply MeteoPrototype.pro]: Yes. This lines was commented in my PR.

File: weather/ui/WeatherIcon.qml (hunk 1 above)
Comment (ValentiWorkLearning) [reply weather/ui/WeatherIcon.qml]: @e1sem I`m not sure, that QML generates  warnings in such cases. But, in C++ code consider to use [[ fallthrough]]; attribute to suppress compiler warnings. 
@Zarrett

File: core/api/WeatherJsonParser.cpp [hunk 4]
```diff
@@ -37,13 +37,23 @@ WeatherData WeatherJsonParser::parseJsonObject(const QJsonObject& data)
 
     auto weatherInfo = data["weather"].toArray().first().toObject();
 
+    QDateTime date = QDateTime::fromMSecsSinceEpoch(static_cast<qint64>(data["dt"].toDouble()*1000));
```
Comment (Stivius) [inline core/api/WeatherJsonParser.cpp]: 1. Magic number
2. Move to separate functions all complex non-trivial actions

File: core/model/QMLWeaterData.hpp [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    void setDayOfWeek(const QString &value);
+    void setCity(const QString &value);
+    void setWeatherIcon(const QString &value);
+    void setWeatherDescription(const QString &value);
+    void setCondition(const QString &value);
+    void setMaxTemperature(const int &value);
+    void setMinTemperature(const int &value);
+    void setPressure(const double &value);
+    void setHumidity(const int &value);
```
Comment (Stivius) [inline core/model/QMLWeaterData.hpp]: Simple types should be without const references

File: core/model/QMLWeaterData.hpp [hunk 6]
```diff
@@ -0,0 +1,80 @@
+#ifndef AWEATHERDATA_H
+#define AWEATHERDATA_H
```
Comment (Stivius) [inline core/model/QMLWeaterData.hpp]: Use #pragma once

File: icons/qt_attribution.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+[
```
Comment (Stivius) [inline icons/qt_attribution.json]: Is this file needed?
